pkg/deploy/devconfig.go

```diff
@@ -117,8 +117,13 @@ func DevConfig(_env env.Core) (*Config, error) {
 			ClusterMDSDConfigVersion:     to.StringPtr(version.DevClusterGenevaLoggingConfigVersion),
 			ClusterMDSDNamespace:         to.StringPtr(version.DevClusterGenevaLoggingNamespace),
 			ClusterParentDomainName:      to.StringPtr(os.Getenv("USER") + "-clusters." + os.Getenv("PARENT_DOMAIN_NAME")),
-			DBTokenClientID:              to.StringPtr(os.Getenv("AZURE_DBTOKEN_CLIENT_ID")),
-			DisableCosmosDBFirewall:      to.BoolPtr(true),
+			CosmosDB: &CosmosDBConfiguration{
+				StandardProvisionedThroughput: 1000,
+				PortalProvisionedThroughput:   400,
+				GatewayProvisionedThroughput:  400,
+			},
+			DBTokenClientID:         to.StringPtr(os.Getenv("AZURE_DBTOKEN_CLIENT_ID")),
+			DisableCosmosDBFirewall: to.BoolPtr(true),
```

Comment:
I agree with both of these changes in a separate fixup effort. The second suggestion would end up changing our actual deployment manifests as that parameter is propagated all the way there, but I do think it would be doable to do that and invert the logic at its final consumption site.

Reply:
+1 to having all the related configuration for CosmosDB in the same spot
+1 this should be a separate effort